src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -25,8 +25,13 @@ public class ClimberSubsystem extends SubsystemBase {
     boolean isClimberEnabled = false;
     double targetVoltage;
     Debouncer sensorlessHomerDebounce = new Debouncer(0.05, Debouncer.DebounceType.kRising);
+    boolean currentlyClimbing = false;
     
 
+    public boolean isCurrentlyClimbing() {
+        return currentlyClimbing;
+    }
+
```

Comment:
great idea to hold a variable in the each subsystem  state, that is totally the right way to do it.

An idea, do we need a separate variable? or could we infer whether the robot is climbing based on reading the target voltage? This could also be done in the shooter subsystem to infer if it is shooting or not. That way we dont need to have an "extra" variable that we need to make sure stays in sync. usually an extra variable makes sense but i think we can just read that variable for this since we are only changing it when we are actually shooting

Reply:
no we dont need but in my opinion its more readable